index.js

```diff
@@ -19,14 +19,26 @@ function parse(req) {
 
     let baton = {
       label: query.label || ((query.compression ? 'gzip ' : '') + 'size'),
-      color: query.color || 'brightgreen',
+      color: query.color || null,
       style: query.style || null,
+      pass: query.pass || query.fail || Infinity,
+      fail: query.fail || query.pass || Infinity,
```

Comment:
I would slightly modify the API.

I would prefer having a `max` (or `limit`) parameter which defines a hard limit. If the size is below `max`, the badge becomes `red`, if not you don't change anything (as the default color is already green).

The idea of having a *blurry* zone where the badge becomes yellow is a good idea, but it should be an opt-in.
Either we could have a `softlimit` or a `threshold` for that. If present and if the size is above `threshold` and below `limit`, then the badge becomes yellow.

What do you think?